native/common/jp_field.cpp

```diff
@@ -49,12 +49,6 @@ void JPField::setStaticField(PyObject *pyobj)
 {
 	JP_TRACE_IN("JPField::setStaticAttribute");
 	JPJavaFrame frame(m_Class->getContext());
-	if (isFinal())
```

Comment:
This condition could never happen?

Reply:
This was a problem with responsibilities.  This condition was never hit  and in tracing I discovered that the PyJPField had already checked against this condition.  Thus this check was redundant.

Reply:
Thanks for the explanation.